deploy/cr.yaml

```diff
@@ -184,6 +185,10 @@ spec:
       mode: slowOp
       slowOpThresholdMs: 100
       rateLimit: 100
+#    loadBalancerSourceRanges:
```

Comment:
I think we need to add it via separate review and as I know we will support it only for mongos

Reply:
ok